### PR TITLE
Add workloadOnly field to helm chart

### PR DESCRIPTION
**Pull request by vincentni**
The new field governs what resource is created for management clusters and workload clusters

### CODE REVIEW

**Comment by TerryHowe** [comment]
when we install this, do we use a different name? Like `eks-anywhere-packages-clusterName`?

**Comment by TerryHowe** [comment]
/hold

Just so people can see this before it merges

**Comment by jonahjon** [comment]
So would this just install a helm chart with just the CRDs?

**Comment by TerryHowe** [comment]
> So would this just install a helm chart with just the CRDs?

The CRDs will already be there.

**Comment by TerryHowe** [comment]
Closes: https://github.com/aws/eks-anywhere-packages/issues/720

**Comment by TerryHowe** [comment]
Is there going to be a follow-up with the secret?

**Review by TerryHowe — APPROVED** [review]
/lgtm

File: charts/eks-anywhere-packages/templates/packagebundlecontroller.yaml
Comment (a-cool-train) [inline charts/eks-anywhere-packages/templates/packagebundlecontroller.yaml:8]: Namespace for management cluster also has the clustername on it.

**Comment by lewisdiamond** [comment]
I think that's fine for now but it highlights the need for a leaner process to push out charts. This should be its own chart given how it excludes almost everything. Given the time constraints we'll go ahead as-is.

**Pull request merged**
